google/cloud/vision/entity.py

```diff
@@ -55,9 +55,7 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = []
-        if 'boundingPoly' in response:
-            bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = Bounds.from_api_repr(response.get('boundingPoly', {}))
```

Comment:
Ok I just added a check in `BoundsBase` to see if the response was `None` that was passed in.